wiki/Tournaments/CWC/2021/zh.md

```diff
@@ -0,0 +1,335 @@
+
+### 2021 年 5 月 3 日，星期一
+
+| A队 | B 队 | 比赛时间 | A 队当地时间 | B 队当地时间 |
+| --: | :-- | :-: | :-: | :-: |
+| 荷兰 ![][flag_NL] | ![][flag_PE] 秘鲁 | **00:30 UTC+08** | 18:30 (星期日) UTC+02 | 11:30 (星期日) UTC-05 |
+| 美国 ![][flag_US] | ![][flag_SE] 瑞典 | **01:00 UTC+08** | 13:00 (星期日) UTC-04 | 19:00 (星期日) UTC+02 |
+| 波兰 ![][flag_PL] | ![][flag_RU] 俄罗斯 | **01:00 UTC+08** | 19:00 (星期日) UTC+02 | 20:00 (星期日) UTC+03 |
+| 法国 ![][flag_FR] | ![][flag_BR] 巴西 | **01:00 UTC+08** | 19:00 (星期日) UTC+02 | 14:00 (星期日) UTC-03 |
+| 芬兰 ![][flag_FI] | ![][flag_PE] 秘鲁 | **02:00 UTC+08** | 21:00 (星期日) UTC+03 | 13:00 (星期日) UTC-05 |
+| 加拿大 ![][flag_CA] | ![][flag_NO] 挪威 | **02:00 UTC+08** | 13:00 (星期日) UTC-05 | 20:00 (星期日) UTC+02 |
+| 德国 ![][flag_DE] | ![][flag_US] 美国 | **02:00 UTC+08** | 20:00 (星期日) UTC+02 | 14:00 (星期日) UTC-04 |
+| 荷兰 ![][flag_NL] | ![][flag_FI] 芬兰 | **03:00 UTC+08** | 21:00 (星期日) UTC+02 | 22:00 (星期日) UTC+03 |
+
+## 图池
+
+### 小组赛
+
+- NoMod
+  - [AAAA vs. Morimori Atsushi - Xrocus (Souvenir) \[Rain\]](https://osu.ppy.sh/beatmapsets/1419078#fruits/2923590)
+  - [Vivid BAD SQUAD - Fragile (Sinnoh) \[Master\]](https://osu.ppy.sh/beatmapsets/1441990#fruits/2966325)
+  - [Akira Complex x Hommarju feat. Yukacco - Connected (WRLD Remix) (Cut ver.) (-MomoX) \[Integrity\]](https://osu.ppy.sh/beatmapsets/1338331#fruits/2966330)
+  - [Reol & nqrse - Ooedo Ranvu (Pho) \[Another\]](https://osu.ppy.sh/beatmapsets/357544#fruits/952576)
+- Hidden
+  - [M2U - Tian Di Kai Pi (ZiRoX) \[Rain\]](https://osu.ppy.sh/beatmapsets/1441999#fruits/2966342)
+  - [Yu\_Asahina - Ongaku -resolve- (Ascendance) \[Mike x Benita's Rain\]](https://osu.ppy.sh/beatmapsets/1442003#fruits/2966346)
+  - [Ariabl'eyeS - Kegare Naki Bara Juuji (Short ver.) (My Angel RangE) \[Left's Insane\]](https://osu.ppy.sh/beatmapsets/535277#fruits/1170869)
+- HardRock
+  - [Studio EIM - Raindrop Flower (F D Flourite) \[Fulaoer\]](https://osu.ppy.sh/beatmapsets/1442018#fruits/2966397)
+  - [USAO - Rampage (Rocma) \[Platter\]](https://osu.ppy.sh/beatmapsets/1442014#fruits/2966392)
+  - [nanobii - rainbow road (Natsu) \[Insane\]](https://osu.ppy.sh/beatmapsets/375073#fruits/823197)
+- DoubleTime
+  - [Eve - As You Like It (GIGACHAD) \[Platter\]](https://osu.ppy.sh/beatmapsets/1442026#fruits/2966405)
+  - [Minato Aqua - Shiroi Yuki no Princess wa (Secre) \[Platter\]](https://osu.ppy.sh/beatmapsets/1375156#fruits/2874509)
+  - [Yorushika - Yuunagi, Bou, Hana Madoi (Delis) \[Insane\]](https://osu.ppy.sh/beatmapsets/1029921#fruits/2154729)
+- Tiebreaker
+  - **[Muzzy - Spectrum (Greaper) \[Refraction\]](https://osu.ppy.sh/beatmapsets/1442175#fruits/2966667)**
+
+## 规则
+
+### 锦标赛规则
+
+1. osu!catch 世界杯是基于国家/地区的团体比赛，在 osu!catch 模式中进行。
+   - 虽然本次比赛计划为 3v3，但可能会根据报名人数而改变。
+2. 分数计算基于 Score V2。
+3. 每轮比赛的图池将由图池管理在实际比赛开始之前的周日公布。选手在比赛过程中只能选择其中的谱面。
+   - 图池包含 [Hidden](/wiki/Game_modifier/Hidden)，[HardRock](/wiki/Game_modifier/Hard_Rock)，[DoubleTime](/wiki/Game_modifier/Double_Time) 以及 NoMod
```

Comment:
- I don't see a need for changing the NoMod to the end.
- The final `.` is missed.

Suggested changes:
> `图池包含 NoMod，[Hidden](/wiki/Game_modifier/Hidden)，[HardRock](/wiki/Game_modifier/Hard_Rock)，[DoubleTime](/wiki/Game_modifier/Double_Time)。`

Reply:
my fault